Sources/protoc-gen-swift/Docs.docc/spm-plugin.md

```diff
@@ -109,18 +114,18 @@ env PROTOC_PATH=/opt/homebrew/bin/protoc xed .
 env PROTOC_PATH=/opt/homebrew/bin/protoc xcodebuild <Here goes your command>
 ```
 
-The other way to point the plugin to the concrete location of the `protoc`
-compiler is by changing the configuration file like this:
+* Point the plugin to the concrete location of the `protoc` compiler is by changing the configuration file like this:
 
 ```json
 {
     "protocPath": "/path/to/protoc",
     "invocations": [...]
 }
-
 ```
 
 > Warning: The configuration file option only solves the problem for leaf packages that are using the Swift package manager
 plugin since there you can point the package manager to the right binary. The environment variable
 does solve the problem for transitive packages as well; however, it requires your users to set
 the variable now. In general we advise against adopting the plugin as a non-leaf package!
+
+* You can start Xcode by running `$ xed .` from the command line from the directory your project is located - this should make `$PATH` visible to Xcode. 
```

Comment:
I think this should be the first option that we list nowadays, it is the easiest to get started with. I think it is worth calling out though that the `protoc` needs to be in the `$PATH` for this to work.